mysql/resource_ti_resource_group.go

```diff
@@ -34,8 +34,7 @@ func (rg *ResourceGroup) buildSQLQuery(prefix string) string {
 	query = append(query, fmt.Sprintf(`PRIORITY = %s`, rg.Priority))
 
 	if rg.QueryLimit != DefaultResourceGroup.QueryLimit {
-		query = append(query, fmt.Sprintf(`QUERY_LIMIT=%s`, rg.QueryLimit))
-
+		query = append(query, fmt.Sprintf(`QUERY_LIMIT=(%s)`, rg.QueryLimit))
```

Comment:
We rely on wrapping the query_limit in parentheses here instead of assuming that it will be wrapped in parentheses by the caller.